### PR TITLE
[ADD] base Seedoo web theme

### PR DESCRIPTION
**Pull request by archetipo**
@angedras @eLBati 
this module adds seedoo graphics web theme, like seedoo official web site.

### CODE REVIEW

File: src/seedoo_theme/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        </script>
+    </head>
+    <body>
+        <!--[if lte IE 8]>
+        <script src="//ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js"></script>
+        <script>CFInstall.check({mode: "overlay"});</script>
+        <![endif]-->
+    </body>
+</html>
+"""
```
Comment (eLBati) [inline src/seedoo_theme/controllers/main.py]: Per essere meno invasivi e non rompere l'ereditarietà, si potrebbe fare
```
controllers.main.html_template = controllers.main.html_template.replace(
    "/web/static/src/img/favicon.ico", "/seedoo_theme/static/src/img/favicon.ico")
```
?

**Comment by eLBati** [comment]
Sostituirei l'attuale src/seedoo_theme/static/src/img/favicon.ico con
http://dfsm9194vna0o.cloudfront.net/1577144-0-logoseedoobig.png

**Comment by eLBati** [comment]
Riguardo al login:
login
Migliorerei il contrasto fra labels (username a password) e sfondo.

File: src/seedoo_theme/controllers/main.py (hunk 1 above)
Comment (archetipo) [inline src/seedoo_theme/controllers/main.py]: Ma qui non cambio solo la favincon, ma anche i title.. non saprei..

Comment (eLBati) [inline src/seedoo_theme/controllers/main.py]: Allora
```
controllers.main.html_template = controllers.main.html_template.replace(
    "/web/static/src/img/favicon.ico", "/seedoo_theme/static/src/img/favicon.ico"
    ).replace("<title>OpenERP</title>", "<title>Seedoo</title>")
```
E' anche più facile leggere quali modifiche stiamo effettuando

**Comment by archetipo** [comment]
@eLBati 
grazie , fatto modifiche

**Comment by eLBati** [comment]
Grazie :+1: 
@angedras aspettiamo un tuo parere o va bene se facciamo subito il merge?

**Comment by angedras** [comment]
Ciao, intanto grazie per il contributo che colma una lacuna non da poco. Passo alle considerazioni di merito:
se guardate i video nella landing di seedoo noterete come i colori che abbiamo usato sono i verdi più scuri del logo, il verde chiaro che è stato usato non mi piace molto sia nella pagina di login, sia nell'intestazione dei menu dentro l'applicativo. Ho visto poi che i colori dei menu principali (ad esempio Comunicazioni, PEC, I miei gruppi nel primo tab Comunicazioni) rimangono blu chiaro come nell'installazione base. Andrebbero cambiati anche quelli per avere una coerenza nell'immagine. Per fare i video avevo utilizzato il modulo web_color_theme che permetteva di modificare questi colori da configurazione, i riferimenti ai css da modificare dovrebbero trovarsi lì.

**Comment by angedras** [comment]
Riapro pr erroneamente chiusa

**Comment by archetipo** [comment]
@angedras  i colori li ho presi da lsito seedoo.it
cmq ok per me np. cambiare colori, piu' che il rif al css mi servirebbe sapere il valore RGB dei color iche avevi usato in modo da riportarli , ed avere il tema coerente con i video.
grazie

**Comment by angedras** [comment]
medio #679B41  scuro #366732

**Comment by archetipo** [comment]
@angedras  ho aggiornato i colori nel css

File: src/seedoo_theme/static/src/css/seedoo.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+.openerp .oe_login .oe_login_logo {
+  top: -116px;
+}
+
+.openerp .oe_topbar {
+  background-color: #366732;
+  background-image: none;
+      font-size: 15px;
+}
+}
```
Comment (angedras) [inline src/seedoo_theme/static/src/css/seedoo.css]: elimina parentesi graffa qui

File: src/seedoo_theme/static/src/css/seedoo.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+.openerp .oe_menu{
+  font-size: 15px!important;
+}
+
+}
```
Comment (angedras) [inline src/seedoo_theme/static/src/css/seedoo.css]: elimina la parentesi anche qui

File: src/seedoo_theme/static/src/css/seedoo.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  background-color: #366732;
+}
+
+.openerp .oe_topbar .oe_topbar_item {
+  color: white;
+}
+
+.openerp .oe_topbar .oe_dropdown_menu li a {
+  color: white;
+}
```
Comment (angedras) [inline src/seedoo_theme/static/src/css/seedoo.css]: Alla fine aggiungi 

.openerp .oe_secondary_menu_section {
  color: #366732;
}

così anche i menu secondari sono ok

**Comment by archetipo** [comment]
@angedras  ri-grazie done

**Comment by angedras** [comment]
:+1:
